### DISCUSSION TITLE
nginx service error

### DISCUSSION THREAD

**Discussion by Prasanthreddy4a5** [learndevopsonline/learndevopsonline#1703] [Q&A] [answered]
Hi,

Whenever I write and run the frontend component, nginx service is giving the below error and I crosschecked the frontend code and everything is fine, can you please help here?
![nginx](https://github.com/learndevopsonline/learndevopsonline/assets/129600823/e53798e1-1e61-45af-a3f1-2634f8a94cfb)

**Comment by sreedharm07** [comment, marked as answer]
did all this components are up and running ?

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/7916d25c-fbda-4e07-ae6c-4c4414ecfb1b)

**Reply by Prasanthreddy4a5** [reply]
No , I am just running the frontend component and found that nginx service is not running even it is installed and enabled.

**Reply by sreedharm07** [reply]
In conf file just mention the dns servers what are only up 
Remaining replace with localhost it will work

**Reply by Prasanthreddy4a5** [reply]
Ok, got it. I will replace it with localhost , Thanks.

Thanks, it worked.

**Comment by Arafique458** [comment]
I noticed in code you are missing some stuff for frontend.
- name: Install Nginx
  ansible.builtin.yum:
    name: nginx
    state: latest
    
    
- name: Start Nginx
  ansible.builtin.systemd:
    state: restarted
    name: nginx
    enabled: true

**Reply by Prasanthreddy4a5** [reply]
I am using them but using dnf instead of yum

- name: Install Nginx
  ansible.builtin.dnf:
      name: nginx
      state: present

- name: Restart Nginx
  ansible.builtin.systemd:
    name: nginx
    state: restarted
    enabled: yes
